src/models/comments_model.ts

```diff
@@ -0,0 +1,28 @@
+import mongoose, { Document, Schema, Types, ObjectId } from "mongoose";
+
+export interface CommentDocument extends Document {
+  _id: ObjectId,
+  postID: Types.ObjectId;
```

Comment:
Are `ObjectId` and `Types.ObjectId` the same?
`_id` and `postID` should be represented with the same type.